tests/test_rx_tx/SConscript

```diff
@@ -27,8 +27,8 @@
 """"""
 import os
 
-# Import('*')
+Import('*')
 
-# Prog('test_rx_tx', Glob("./test_rx_tx.cpp"), with_libs='utils::utils',
```

Comment:
This test might be broken? Does it compile?

Reply:
It does compile, which is why I uncommented it. I did not try running it though.